client/signup/steps/woocommerce-install/complete.tsx

```diff
@@ -0,0 +1,39 @@
+import { BackButton, NextButton, SubTitle, Title } from '@automattic/onboarding';
+import { useI18n } from '@wordpress/react-i18n';
+import debugFactory from 'debug';
+import { ReactElement } from 'react';
+import { GoToStep } from '../../types';
+
+const debug = debugFactory( 'woop:complete' );
+
+interface Props {
+	goToStep: GoToStep;
+}
+
+export default function Complete( { goToStep }: Props ): ReactElement | null {
+	const { __ } = useI18n();
+
+	debug( 'rendering step' );
+
+	return (
+		<>
+			<div className="woocommerce-install__heading-wrapper">
+				<div className="woocommerce-install__heading">
+					<Title>{ __( 'Setup Complete' ) }</Title>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **32** times:
`translate( 'Complete setup' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/woo-installer)